services/api/src/middlewares/validate.js

```diff
@@ -5,7 +5,7 @@ function validate(schemas, options) {
     abortEarly: false
   };
 
- return async function Validate(ctx, next) {
+ const promise = async function Validate(ctx, next) {
```

Comment:
this isn't a promise but just a function no?
You should be able to do `async function Validate` and later `Validate.schemas`